sync/class.jetpack-sync-module-posts.php

```diff
@@ -81,7 +81,9 @@ function filter_post_content_and_add_links( $post_object ) {
 		$post->post_excerpt_filtered   = apply_filters( 'the_content', $post->post_excerpt );
 		$post->permalink               = get_permalink( $post->ID );
 		$post->shortlink               = wp_get_shortlink( $post->ID );
-		$post->dont_email_post_to_subs = get_post_meta( $post->ID, '_jetpack_dont_email_post_to_subs', true );
+		$post->dont_email_post_to_subs = in_array( 'subscriptions', Jetpack::get_active_modules() ) ?
```

Comment:
Perhaps `Jetpack::is_module_active( 'subscriptions' )` can be used here instead?